README.md

```diff
@@ -12,7 +12,7 @@ PHP client library for [Coveralls](https://coveralls.io).
 
 # Prerequisites
 
-- PHP 5.3 or later
+- PHP 5.5+ for 2.x or 5.2+ for 1.x
```

Comment:
Assuming I was on my working env with git and not tablet ;)
And yes, I hate to edit files directly on github ;)